lib/cli.js

```diff
@@ -14,6 +14,7 @@ module.exports = CLI
   .option('-p, --patch', 'create a patch changelog')
   .option('-m, --minor', 'create a minor changelog')
   .option('-M, --major', 'create a major changelog')
+  .option('-t, --tag [range]', 'generate from specific tag', '')
```

Comment:
Can you change it from `[range]` to `<range>`? That will indicate that if you specify `-t`, a range is [a required argument](https://www.npmjs.com/package/commander#specify-the-argument-syntax). And then you can remove the default empty string. If someone is using the flag, they should be passing something in.

Also, can you add an example of a normal tag and then a range in the description? `generate from specific tag or range e.g. v1.2.3, v1.2.3..v1.2.4`